backend/database_handler/snapshot_manager.py

```
@@ -70,8 +70,9 @@ def create_snapshot(self) -> Snapshot:
                 "appeal_undetermined": tx.appeal_undetermined,
                 "triggered_by_hash": tx.triggered_by_hash,
                 "appealed": tx.appealed,
```

Ruff (0.11.9)

Dictionary key literal `"appealed"` repeated

Remove repeated key literal `"appealed"`

(F601)